cmd/maya-apiserver/cstor-operator/spc/controller.go

```diff
@@ -188,6 +196,10 @@ func (c *Controller) deleteSpc(obj interface{}) {
 			return
 		}
 	}
+	if spc.Labels[string(apis.OpenEBSUpgradeKey)] == "true" {
```

Comment:
logic should be handled at informer code

Reply:
same as above comment